src/evaluation.py

```diff
@@ -139,49 +211,53 @@ def solve_task(task):
 
 
 def find_frame_url(project_name, driver):
```

Comment:
Should this function be something else? I am not sure if it is returning a URL.

Reply:
Yeah, I need to change its name